apps/channels/forms.py

```diff
@@ -18,6 +18,8 @@
 
 
 class ChannelForm(forms.ModelForm):
+    name = forms.CharField(required=False, help_text="If you leave this blank, it will default to the experiment name")
```

Comment:
Maybe add a placeholder: `widget=forms.TextInput(attrs={'placeholder': 'Channel Name'})`

(it would be nice if this was the experiment name)

Reply:
https://github.com/dimagi/open-chat-studio/pull/823/commits/b62b2c6f0e28124412a10cbbfd4b799e5df1fb91